toplevel/toploop.ml

```diff
@@ -185,7 +185,7 @@ let loop ppf =
   Clflags.debug := true;
   Location.formatter_for_warnings := ppf;
   if not !Clflags.noversion then
-    fprintf ppf "        OCaml version %s%s%s@.@."
+    fprintf ppf "  OCaml version %s%s%s    #help;; for help@.@."
```

Comment:
I'm not sure why the spacing was altered? I like the message, but why not on its own line?
```suggestion
    fprintf ppf "        OCaml version %s%s%s@.@.Type #help;; for help@.@."
```

cf. utop:
```
─────────────┬─────────────────────────────────────────────────────────────┬──────────────
             │ Welcome to utop version 2.8.0 (using OCaml version 4.11.2)! │
             └─────────────────────────────────────────────────────────────┘

Type #utop_help for help about using utop.
```

Reply:
I trimmed leading spaces so the message would fit in 80 columns with long versions such as `4.14.0+dev10-2021-06-03`.  
  
I just liked the compact one line look. I also took the phrasing from GHCi:

    GHCi, version 8.6.5: http://www.haskell.org/ghc/  :? for help
    Prelude>